spec/shr_finding_map_dstu2.txt

```diff
@@ -5,7 +5,7 @@ Target:		FHIR_DSTU_2
 
```

Comment:
missing a DSTU2 mapping to DiagnosticReport.  An STU3 mapping however exists.  But for some reason, oncocore-config.json specifies DSTU2 and even though it's missing from shr-finding_map_dstu2, it still appears in the generated IG.